DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Plugin/RevitExternalApplication.cs

```diff
@@ -69,6 +83,35 @@ public Result OnStartup(UIControlledApplication application)
     return Result.Succeeded;
   }
 
+  /// <summary>
+  /// If we directly use ex.IsFatal() in the above method, and there is an outdated version of Core loaded,
+  /// the MissingMethodException will be throw prior to executing the OnStartup method. We would prefer if
+  /// the exception is thrown after the <see cref="DllConflictManager.DetectConflicts"/> method is executed
+  /// </summary>
+  /// <param name="ex"></param>
+  /// <returns></returns>
+  private static bool IsFatal(Exception ex) => ex.IsFatal();
```

Comment:
Not sure this is the best way to go about this. Maybe we should just decouple the conflict detection from any Speckle related stuff and have it run before anything else, but maybe I'm missing something here 🤔 

Right now, conflict resolution lives in a DLL that depends on Core, which inevitably leads to us having issues detecting conflicts in Core itself. I've done this type of things in the past on Grasshopper and they don't end up aging well 🥲

Reply:
Gotcha. Yeah this makes sense.